### ISSUE TITLE
mean and std

### ISSUE THREAD

**Issue by hao522** [ericsun99/Shufflenet-v2-Pytorch#1]
what's the mean and std for those pretrained models you provided

**Comment by ericsun99** [comment]
pls take a look at line 72 in eval.py.
normalize = transforms.Normalize(mean=[0.485, 0.456, 0.406],
                                     std=[0.229, 0.224, 0.225])
